seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -949,7 +1015,7 @@ angular.module('BE.seed.controller.inventory_list', [])
       function currentColumns () {
         // Save all columns except first 3
         var gridCols = _.filter($scope.gridApi.grid.columns, function (col) {
-          return !_.includes(['treeBaseRowHeaderCol', 'selectionRowHeaderCol', 'notes_count', 'merged_indicator', 'id'], col.name) && col.visible;
+          return !_.includes(['treeBaseRowHeaderCol', 'selectionRowHeaderCol', 'notes_count', 'merged_indicator', 'id'], col.name) && col.visible && col.id;
```

Comment:
this turned out to be an existing bug, fixed here (screenshot from develop branch)
![Screen Shot 2021-06-15 at 9 58 06 AM](https://user-images.githubusercontent.com/18518728/122101712-e17f3000-cdd1-11eb-9ff4-3cb1f9c16e7f.png)